packages/editor/src/components/post-type-support-check/test/index.js

```diff
@@ -10,27 +10,26 @@ import { PostTypeSupportCheck } from '../';
 
 describe( 'PostTypeSupportCheck', () => {
 	it( 'renders its children when post type is not known', () => {
-		let postType;
-		const tree = create(
-			<PostTypeSupportCheck postType={ postType } supportKeys="title">
+		const { container } = render(
```

Comment:
Cleanup as part of the suggestion below.

```suggestion
		render(
```